charts/spire/charts/spire-server/templates/controller-manager-cluster-ids.yaml

```diff
@@ -5,7 +5,6 @@ apiVersion: spire.spiffe.io/v1alpha1
 kind: ClusterSPIFFEID
 metadata:
   name: {{ include "spire-controller-manager.fullname" $root }}-service-account-based
-  namespace: {{ include "spire-server.namespace" $root }}
```

Comment:
Noticed during development that the namespace was specified, but the object is cluster wide so namespace is always ignored. Removed to be consistent with the newly added objects.